.github/ISSUE_TEMPLATE/page-report.yml

```
@@ -0,0 +1,55 @@
+name: "[via MDN pages only]"
+description: Issue filed via "Report a problem with this content on GitHub" link on MDN pages.
+labels: ["via page report", "needs triage"]
+body:
```

```suggestion
body:
  - type: input
    id: url
    attributes:
      label: MDN URL
      validations:
        required: true
```
- Not having the MDN URL end up at the top of the created issue would be a regression over what we have now.
- Having an MDN URL at the top of every issue helps contributors who are looking through open issues to try to find ones they can contribute patches for, and also helps us when we’re doing our own issue triage.
- Otherwise, without the MDN URL at the top, contributors and we ourselves would need to scroll down all the way to the metadata section at the end of the issue to find the MDN URL.
- Yari can pass the MDN URL to this form in the link URL as a query param; e.g.:
https://github.com/ddbeck/issue-form-testing/issues/new?template=page-report.yml&url=https://developer.mozilla.org/en-US/docs/Web/API/Navigator/platform

---

Thanks for this, @sideshowbarker. I'll return the dedicated URL field (though after the introductory help text).